src/routes/(app)/+layout.svelte

```diff
@@ -8,6 +8,7 @@
   import Banners from '$lib/components/banner/banners.svelte';
   import DataEncoderSettings from '$lib/components/data-encoder-settings.svelte';
   import SideNavigation from '$lib/components/side-nav.svelte';
+  import SkipNav from '$lib/components/skip-nav.svelte';
```

Comment:
Can I be pendantic and ask that we call this `SkipNavigation`?

Reply:
[Done](https://github.com/temporalio/ui/pull/1611/commits/741522d878311c87d8d05dbdf699407f1f6e15db) ✅